src/DataViews/include/DataViews/DataView.h

```diff
@@ -224,7 +224,7 @@ class DataView {
   int sorting_column_ = 0;
   std::string filter_;
   int update_period_ms_;
-  absl::flat_hash_set<int> selected_indices_;
+  absl::flat_hash_set<uint64_t> selected_indices_;
```

Comment:
Shouldn't `GetContextMenuWithGrouping` and `OnRefresh` also take a vector of `uint64_t` ?

Reply:
Ah right, I reverted to `int` actually, and added a static cast instead. Thanks.